source/systems/gaea_user_guide.rst

```diff
@@ -398,6 +398,61 @@ For example:
 
 Once the job is finished, it should produce an output file.
 
+Displaying Graphics on Compute Nodes
+====================================
+
+.. note::
+
+  Forwarding the display from the compute nodes is an extremely slow process,
+  as three hops are needed to get back to the user's local system. You should
+  only forward from the login nodes.
+
+Follow these steps to display graphics from the compute node.
+
+First terminal
+--------------
+
+Log in to the Gaea system:
+
+.. code-block:: shell
+
+  $ ssh <username>@gaea-rsa.princeton.rdhpcs.noaa.gov
```

Comment:
Should we be specifying the princeton bastion here, rather than the load balancer `gaea-rsa.rdhpcs.noaa.gov`?

Reply:
The hostname does need to be generic, and not just the princeton bastion.  (It could be the CAC bastion as well.)

Reply:
changed to gaea-rsa.rdhpcs.noaa.gov